src/docs/ftw-content/hosted-translations/index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Hosted translations with Asset Delivery API
+slug: hosted-translations
+updated: 2022-05-13
+category: ftw-content
+ingress:
+  This article describes how hosted translations work in Flex Template
+  for Web (FTW).
+published: true
+---
+
+Starting from 2022-05, FTW template translations can be managed both in
+the built-in translation files and in Flex Console. This article
+describes how the FTW template uses the hosted translations and merges
+them with the built-in translations
+
+_**Note:** If you want to implement this feature into your pre-version
+XX.XX FTW template, you can see the necessary modifications in the PRs
```

Comment:
Related to this, I think the next version are:
- FTW-daily v8.5
- FTW-hourly v10.5
- FTW-product: v9.2

aka minor version update

Reply:
Thanks, added this!